libevmasm/Assembly.cpp

```diff
@@ -396,7 +399,8 @@ void Assembly::assemblyStream(
 {
 	Functionalizer f(_out, _prefix, _sourceCodes, *this);
 
-	for (auto const& i: m_items)
+	// TODO: support EOF
```

Comment:
In cases like this (and e.g. similarly for ``Assembly::assemblyJSON`` and possibly a few of other similar TODOs like this), it'd be better to add an explicit

```c++
solUnimplementedAssert(!m_eofVersion.has_value(), "Assembly output for EOF is not yet implemented.");
```

Better to error out and then implement it properly later in a subsequent PR than to give a partial output of only the first code section with the rest missing.

Reply:
Done.